design-proposals/klip-43-timestamp-data-type-support.md

```diff
@@ -52,21 +52,21 @@ CREATE STREAM stream_name (time TIMESTAMP, COL2 STRING) AS ...
 CREATE TABLE table_name (col1 STRUCT<field TIMESTAMP>) AS ...
 ```
 
-TIMESTAMPS will be displayed in console as strings in ODBC canonical format with millisecond precision:
+TIMESTAMPS will be displayed in console as strings in ISO-8601 form with millisecond precision:
 
 ```roomsql
 > SELECT time FROM stream_name EMIT CHANGES;
 
 +------------------------+
 |time                    |
 +------------------------+
-|1994-11-05 13:15:30:112 |
+|1994-11-05T13:15:30:112 |
```

Comment:
```suggestion
|1994-11-05T13:15:30.112 |
```

There is a fractional part separated by a dot mentioned in the standard: `"yyyy-mm-ddThh:mm:ss[.mmm]`.